src/Controls/src/Xaml/SimplifyOnPlatformVisitor.cs

```diff
@@ -8,10 +9,37 @@ class SimplifyOnPlatformVisitor : IXamlNodeVisitor
 	{
 		public SimplifyOnPlatformVisitor(string targetFramework)
 		{
-			TargetFramework = targetFramework;
+
+			if (string.IsNullOrEmpty(targetFramework))
+				return;
+
+#if NETSTANDARD2_0
+			if (targetFramework.Contains("-android"))
+#else
+			if (targetFramework.Contains("-android", StringComparison.Ordinal))
```

Comment:
If you wanted, I think `IndexOf("-android", StringComparison.OrdinalIgnoreCase) != -1` should work for `NETSTANDARD2_0` and you could remove `#if`.